apps/native-component-list/src/screens/Amplitude/TrackingOptionsSelector.tsx

```diff
@@ -0,0 +1,263 @@
+        }, 5000);
+      }
+    );
+  };
+
+  render() {
+    return (
+      <View style={{ marginTop: 20 }}>
+        <Button title="Toggle tracking options list" onPress={this._toggleListVisibility} />
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Adid',
+            valueName: 'disableAdid',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Carrier',
+            valueName: 'disableCarrier',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable City',
+            valueName: 'disableCity',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Country',
+            valueName: 'disableCountry',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable DMA',
+            valueName: 'disableDMA',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Device brand',
+            valueName: 'disableDeviceBrand',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Device manufacturer',
+            valueName: 'disableDeviceManufacturer',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Device model',
+            valueName: 'disableDeviceModel',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable IDFV',
+            valueName: 'disableIDFV',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable IP Address',
+            valueName: 'disableIPAddress',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Language',
+            valueName: 'disableLanguage',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Geo?',
+            valueName: 'disableLatLng',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable OS Name',
+            valueName: 'disableOSName',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable OS version',
+            valueName: 'disableOSVersion',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Platform',
+            valueName: 'disablePlatform',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable Region',
+            valueName: 'disableRegion',
+          })}
+        {this.state.showList &&
+          this._renderToggle({
+            title: 'Disable version name',
+            valueName: 'disableVersionName',
+          })}
```

Comment:
Use [`Fragments`](https://reactjs.org/docs/fragments.html#short-syntax) and reduce boilerplate:
```suggestion
        {this.state.showList && (
          <>
            {
              [
            	{ title: 'Disable Adid', valueName: 'disableAdid' },
            	{ title: 'Disable Carrier', valueName: 'disableCarrier' },
            	{ title: 'Disable City', valueName: 'disableCity' },
            	{ title: 'Disable Country', valueName: 'disableCountry' },
            	{ title: 'Disable DMA', valueName: 'disableDMA' },
                { title: 'Disable Device brand', valueName: 'disableDeviceBrand' }
            	{ title: 'Disable Device manufacturer', valueName: 'disableDeviceManufacturer' },
            	{ title: 'Disable Device model', valueName: 'disableDeviceModel' },
            	{ title: 'Disable IDFV', valueName: 'disableIDFV' },
            	{ title: 'Disable IP Address', valueName: 'disableIPAddress' },
            	{ title: 'Disable Language', valueName: 'disableLanguage' },
            	{ title: 'Disable Geo?', valueName: 'disableLatLng' },
                { title: 'Disable OS Name', valueName: 'disableOSName' },
                { title: 'Disable OS version', valueName: 'disableOSVersion' },
                { title: 'Disable Platform', valueName: 'disablePlatform' },
                { title: 'Disable Region', valueName: 'disableRegion' },
                { title: 'Disable version name', valueName: 'disableVersionName' },
              ].map(this._renderToggle)
            }
          </>
        )}
```
You can extract these toggles definitions into some meaningful variable 😉